src/interface/guide/components/Preparation/PreparationSection.tsx

```diff
@@ -24,10 +24,12 @@ const PreparationSection = ({
 }: Props) => (
   <Section title="Preparation">
     <EnchantmentSubSection recommendedEnchantments={recommendedEnchantments} />
-    <EnhancementSubSection
-      recommendedLegEnhancements={recommendedLegEnhancements}
-      recommendedWeaponEnhancements={recommendedWeaponEnhancements}
-    />
+    {expansion !== Expansion.WrathOfTheLichKing && (
```

Comment:
I recommend using `isRetailExpansion` here: https://github.com/WoWAnalyzer/WoWAnalyzer/blob/dragonflight/src/game/Expansion.ts#L23
```suggestion
    {isRetailExpansion(expansion) && (
```